mlflow/gateway/constants.py

```diff
@@ -1,3 +1,4 @@
 MLFLOW_GATEWAY_HEALTH_ENDPOINT = "/health"
-MLFLOW_GATEWAY_ROUTE_BASE = "/api/2.0/gateway/routes/"
+MLFLOW_GATEWAY_CRUD_ROUTE_BASE = "/api/2.0/gateway/routes/"
+MLFLOW_GATEWAY_ROUTE_BASE = "/gateway/routes/"
```

Comment:
@BenWilson2 on Databricks, this is just `/gateway/`. Can we update it in OSS for consistency too? 

```
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://e2-dogfood.staging.cloud.databricks.com/gateway/routes/corey-test/invocations. Response text: {"error_code":"ENDPOINT_NOT_FOUND","message":"ENDPOINT_NOT_FOUND: Path must be of form /gateway/<route>/invocations"}
```